### PR TITLE
feat(components): menu item support secondary text

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: libs/components/src/lib/menu-item/menu-item.ts [hunk 1]
```diff
@@ -16,6 +16,14 @@ export class MenuItem extends FastMenuItem {
...
 	 */
 	@attr text?: string;
+	/**
+	 * Indicates the menu item's secondary text.
+	 *
+	 * @public
+	 * @remarks
+	 * HTML Attribute: text
+	 */
+	@attr({ attribute: 'secondary-text' }) secondaryText?: string;
```
Comment (yinonov) [inline libs/components/src/lib/menu-item/menu-item.ts:26]: should this be ordered by category first?
```suggestion
	@attr({ attribute: 'text-secondary' }) textSecondary?: string;
```

Comment (yinonov) [inline libs/components/src/lib/menu-item/menu-item.ts:26]: should this be ordered by category first?
```suggestion
	@attr({ attribute: 'text-secondary' }) textSecondary?: string;
```

File: libs/components/src/lib/menu-item/README.md [hunk 2]
```diff
@@ -17,6 +17,39 @@ Represents a menu-item custom element.
...
+
+Use the `text-secondary` attribute (or `textSecondary` property) to set the menu item's secondary text.
+
+Note: to improve readability, **avoid long text and multiple lines** where possible.
+
+```html preview
+<style>
+ :root {
+  height: 500px;
+ }
```
Comment (rachelbt) [inline libs/components/src/lib/menu-item/README.md:46]:
```suggestion

```
why do we need this?

File: libs/components/src/lib/menu-item/README.md [hunk 3]
```diff
@@ -29,7 +62,7 @@ set `role` to change the role of the menu item
   html { 
     block-size: 360px; 
   }
-</style>
+</>
```
Comment (rachelbt) [inline libs/components/src/lib/menu-item/README.md:65]: something is broken here and there's no preview:
image

**Review by rachelbt — COMMENTED** [review]
didn't manage to control the width of the menu-item

**Pull request merged**